lib/embed.js

```diff
@@ -1,19 +1,42 @@
 import renderTextItems from './text-items';
+const twitterRegex = /^https?:\/\/twitter\.com\/([-a-zA-Z0-9+&@#%?=~_|!:,.;]+)\/status(es){0,1}\/(\d+)/;
+
+function formatTwitterUrl (url) {
+  const match = url.match(twitterRegex);
+  if (!match) {
+    return '';
+  }
+
+  return `https://twitter.com/${match[1]}/status/${match[3]}`;
+}
```

Comment:
@iefserge let's go with this now, we should probably normalize urls a bit harder on an article-json level but that can be done at a later stage.